catkit2/utils.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+def rotate_and_flip_image(data, theta, flip):
+    """
+    Converts an image based on rotation and flip parameters.
+    :param data: Numpy array of image data.
+    :param theta: Rotation in degrees of the mounted camera, only these discrete values {0, 90, 180, 270}
+    :param flip: Boolean for whether to flip the data using np.fliplr.
+    :return: Converted numpy array.
```

Comment:
We use the numpy docstring format (some non-compliant docstrings might exist in the repos, but we try to get rid of them) - could you change to that format?
https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Can do!